### PR TITLE
Upgrade ruby from 2.7.0 to 2.7.2

### PR DESCRIPTION
**Pull request by ajsmth**
# Why

From what I can understand -- ruby 2.7.0 doesn't compile w/ arm, but ruby 2.7.2 does - here is the output from trying to install 2.7.0 on a M1 machine: 

Screen Shot 2022-03-14 at 9 43 44 AM

This has been a long standing issue w/ M1 Macs - I usually just manually bump this file and don't commit it with my PRs, but I figured as adoption for M1 increases this might be something we want to address 

# How

Bump the version in `ruby-version`

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by tsapeta — APPROVED** [review]
Cool 👍 Client job on Android is failing, but it doesn't seem to be caused by Ruby upgrade. I already retried the job, but it failed too.

**Comment by ajsmth** [comment]
my CI luck is always bad

**Review by nicknovitski — APPROVED** [review]
The latest ruby 2.7.x is 2.7.5, and it would be fine to use that, or an even newer one, as long as the tools supported it.

If we're sure that the CI failure isn't related to the change, I don't object to merging it.  But does this check fail often?

**Comment by tsapeta** [comment]
Actually, at second glance I think it might be related. Take a look at the stacktrace at the end — you can find both Ruby versions in those paths 🤔

**Review by nicknovitski — CHANGES_REQUESTED** [review]
Then it looks like something else needs to change in the android gha workflow to support this, maybe in how it's using the setup-ruby action?  Or is fastlane itself being setup to expect the old version?

**Comment by ajsmth** [comment]
seems to be fixed now?

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.7.0
+2.7.2
```
Comment (tsapeta) [inline .ruby-version:1]:
```suggestion
2.7.6

```

**Review by nicknovitski — APPROVED** [review]
Great!

**Review by tsapeta — APPROVED** [review]
Would you mind updating to the newest 2.x version? That is `2.7.6` now. It introduces mostly just security fixes since `2.7.2`.

**Pull request merged**